app/views/projects/samples/metadata/_table.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="table-container">
+  <table
+    class="w-full text-sm text-left text-slate-500 dark:text-slate-400 mb-4"
+    data-controller="selection"
+    data-selection-action-link-outlet=".action-link"
+    id="metadata-table"
+  >
+    <thead class="text-slate-700 bg-slate-50 dark:bg-slate-900 dark:text-slate-400">
+      <tr>
+        <th scope="col" class="px-6 py-3"><%= t("projects.samples.show.table_header.metadata_field") %></th>
```

Comment:
Let's update `Metadata Field` header to `Key`

Reply:
Fixed in 0fb8fc1c66eb29d81ced91306361fadc7cb1d8b3